docs/reference/pages/theory.md

```diff
@@ -104,10 +104,16 @@ The basic workflow is as follows:
 
 Note that you have to click _Publish_ after setting the _go-live at_ field for the revision to be scheduled. Saving a draft revision with the _go-live at_ field without clicking _Publish_ will not schedule it to be published.
```

Comment:
Maybe we should make this a note admonition also, we do try to avoid these being over used but as it already says 'note', maybe we should make it one.

Reply:
I agree, in this case its appropriate